src/ape/managers/project/manager.py

```diff
@@ -447,16 +447,23 @@ def _get_contract(self, name: str) -> Optional[ContractContainer]:
 
         return None
 
-    # @property
-    # def meta(self) -> PackageMeta:
-    #     return PackageMeta(**self.config_manager.get_config("ethpm").serialize())
+    @property
+    def meta(self) -> PackageMeta:  # type: ignore
```

Comment:
probably put the type ignore on the return stmt

Reply:
I don't think this fixed the type issue.